res/controllers/midi-controller-api.d.ts

```diff
@@ -0,0 +1,32 @@
+
+/** MidiControllerJSProxy */
+
+declare namespace controller {
```

Comment:
I figured out why my intellisense was not working. I always tried to test it with `midi.sendShortMsg` and `midi` was not recognized because the namespace is misdeclared here.
```suggestion
declare namespace midi {
```